staging/dex-k8s-authenticator/html-templates/kubectl-plugin-instructions.html

```diff
@@ -0,0 +1,112 @@
+<h1>Build kubeconfig with kubectl</h1>
+{{ if .caPem }}
+    <h2>Copy Cluster CA Certificate</h2>
+    <p>This is the CA certificate needed to validate connections to your cluster. If your cluster is using a well known
+        certificate authority, such as Let's Encrypt, or you have added your CA certificate to your system certificate pool;
+        you may skip this section.</p>
+    <div class="command">
+
+        <button class="btn" style="float:right" data-clipboard-snippet="">
+            <img class="clippy" width="13" src="{{ .webPrefixPath }}static/clippy.svg" alt=""/>
+        </button>
+        <pre><code>mkdir -p ${HOME}/.kube/certs/{{ .clusterName }}/ &amp;&amp; cat &lt;&lt; EOF &gt; ${HOME}/.kube/certs/{{ .clusterName }}/k8s-ca.crt
+{{ .caPem }}
+EOF</code></pre>
+    </div>
+{{ end }}
+
+<h2>Download and Install the Konvoy credentials plugin</h2>
+<p>The plugin is currently available for Linux and MacOSX. Use the following command appropriate
+  for your operating system< to download and install the plugin</p>
+<h3>OSX:</h3>
+<div class="command">
+  <button class="btn" style="float:right" data-clipboard-snippet="">
+    <img class="clippy" width="13" src="{{ .webPathPrefix }}static/clippy.svg" alt=""/>
+  </button>
+{{ if .caPem }}
+  <pre><code>curl --create-dirs --cacert ${HOME}/.kube/certs/{{ .clusterName }}/k8s-ca.crt -sL {{ .darwinURL }} \
+      -o ${HOME}/{{ .installPath }} && \
+      chmod +x ${HOME}/{{ .installPath }}
+    </code></pre>
+{{ else }}
+    <pre><code>curl --create-dirs {{ .darwinURL }} \
+      -o ${HOME}/{{ .installPath }} && \
+      chmod +x ${HOME}/{{ .installPath }}
+    </code></pre>
+{{ end }}
+</div>
+<h3>LINUX:</h3>
+<div class="command">
+  <button class="btn" style="float:right" data-clipboard-snippet="">
+    <img class="clippy" width="13" src="{{ .webPathPrefix }}static/clippy.svg" alt=""/>
+  </button>
+    {{ if .caPem }}
+        <pre><code>curl --create-dirs --cacert ${HOME}/.kube/certs/{{ .clusterName }}/k8s-ca.crt -sL {{ .linuxURL }} \
+      -o ${HOME}/{{ .installPath }} && \
+      chmod +x ${HOME}/{{ .installPath }}
+    </code></pre>
+    {{ else }}
+        <pre><code>curl --create-dirs {{ .linuxURL }} \
+      -o ${HOME}/{{ .installPath}}} && \
+      chmod +x ${HOME}/{{ .installPath }}
+    </code></pre>
+    {{ end }}
+</div>
+
+<h2>Configure kubect to Use the Plugin</h2>
```

Comment:
Questionable about the title casing of kubectl, but we should be consistent with whatever you chose above.

```suggestion
<h2>Configure kubectl To Use The Plugin</h2>
```